Makefile

```diff
@@ -14,6 +14,14 @@ export PATH := $(GOBIN):$(PATH)
 GOFMT := $(GOBIN)/gofmt
 GO := $(GOBIN)/go
 
+CLUSTER_DIR ?= cluster
```

Comment:
We don't need this change. All the scripts are already in the ./cluster/ directory. Is there any reason to parametrize those?

Reply:
just because it's used more than once, then I added it for easier future changes.